essentials-framework/Essentials Core/PepperDashEssentialsBase/Secrets/Interfaces.cs

```diff
@@ -0,0 +1,24 @@
+﻿using PepperDash.Core;
+
+namespace PepperDash.Essentials.Core
+{
+    /// <summary>
+    /// All ISecrecretProvider classes must implement this interface.
+    /// </summary>
+    public interface ISecretProvider : IKeyed
+    {
+        void SetSecret(string key, object value);

```

Comment:
As mentioned above, change the signature to be `bool SetSecret(string key, object value)`